modules/level_set/test/tests/reinitialization/tests

```diff
@@ -1,13 +1,23 @@
 [Tests]
-  issues = '#8465'
   design = 'level_set/index.md'
-  [./full]
-    # Runs two steps of a full reinitialization problem just to make sure it continues to work
+  [./full_original_reinitialization]
+    # Runs two steps of a full original reinitialization problem just to make sure it continues to work
     type = Exodiff
     input = master.i
-    exodiff = master_out.e
-    cli_args = Executioner/num_steps=2
+    exodiff = master_original.e
+    cli_args = 'Executioner/num_steps=2 Outputs/file_base=master_original'
     method = 'opt' # complete reinitilization solve, debug is slow and not needed
-    requirement = "The level set module shall be capable of solving the level set equation with reinitialization."
+    requirement = "The level set module shall be capable of solving the level set equation with original reinitialization."
+    issues = '#14849 #'
+  [../]
+  [./full_modified_reinitialization]
+    # Runs two steps of a full modified reinitialization problem just to make sure it continues to work
+    type = Exodiff
+    input = master.i
+    exodiff = master_modified.e
+    cli_args = 'Executioner/num_steps=2 MultiApps/reinit/input_files=reinit_modified.i Outputs/file_base=master_modified'
+    method = 'opt' # complete reinitilization solve, debug is slow and not needed
```

Comment:
FYI - I prefer to see '!dbg' when possible. The reason is that some developers like to work in "devel" or "oprof" modes and this test would still run.

Reply:
done!